test/fixtures/tables/NDB_BVL_FeedbackTest.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0"?>
+<mysqldump xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<database name="LorisTest">
```

Comment:
It does not have to match what is in the config.xml. This DB name is not used by DBUnit. It's the DB name from Loris_`PHPUnit_Database_TestCase::getConnection` which is used when populating tables.